openedx/core/djangoapps/content/block_structure/tasks.py

```diff
@@ -22,9 +30,14 @@ def update_course_in_cache(course_id):
     try:
         course_key = CourseKey.from_string(course_id)
         api.update_course_in_cache(course_key)
+    except NO_RETRY_TASKS as exc:
+        log.info("update_course_in_cache encountered unrecoverable error: {}".format(exc))
+        raise
+    except RETRY_TASKS as exc:
+        log.info("update_course_in_cache encounted expected error, retrying.")
```

Comment:
Change this to `log.exception` so the stack trace of the original exception gets logged.